1.7/administration/security-and-authentication/auth-api.md

```diff
@@ -0,0 +1,47 @@
+---
+post_title: Authentication HTTP API Endpoint
+layout: page
+---
+You can make external calls to HTTP API endpoints in your DC/OS cluster.
+
+You must first obtain an authentication token and then include it in your HTTP request. 
+
+By default authentication tokens expire after 5 days. You can view the expiration time in the ["exp" (Expiration Time) Claim](https://tools.ietf.org/html/rfc7519#section-4.1.4) of the JSON Web Token (JWT). You can refresh your JWT by re-logging in to DC/OS.
+
+# Generate the authentication token
+
+### Request
+
+Log in with a POST request against `acs/api/v1/auth/login`.
+
+### Data
+
+Specify the user ID (`<user-id>`), password (`<password>`), and external hostname (`<master-host-name>`):
```

Comment:
For DC/OS, for now, I suggest we tell people to use the CLI to do an auth login and then get the token from there.

In 1.8 or 1.9 we'll have service accounts which will make this easier.